tests/src/Kernel/Access/WikiTest.php

```diff
@@ -204,15 +225,6 @@ public function entityOperationAccessProvider() {
         ],
       ],
       [
-        // Non-members should have the right to edit any group content.
```

Comment:
@dakala you have removed this part for now, just because the tests failed, right?